packages/cli/src/bin/pipcook-daemon.ts

```diff
@@ -36,7 +36,7 @@ async function start(): Promise<void> {
     if (message.event === 'ready') {
       daemon.disconnect();
       daemon.unref();
-      logger.success(`Pipcook is on http://localhost:${message.data.listen}`);
+      logger.success('Pipcook service started, please visit https://pipboard.vercel.app');
```

Comment:
I think the daemon url should not be removed, if we access daemon from remote by `@pipcook/pipcook-cli`, like:
```sh
$ pipcook job list -h 192.168.1.33 -p 1234
```
The `port` will miss.

Reply:
Now visit [http://localhost:6927/](http://localhost:6927/) and 404 will appear. suggest port can be viewed on pipboard web.